web/src/frontend/src/digisos/skjema/ettersendelse/ettersendelse.tsx

```diff
@@ -122,96 +120,83 @@ class Ettersendelse extends React.Component<Props, OwnState> {
 	}
 
 	render() {
-		const { originalSoknad, ettersendelser, visEttersendelse } = this.props;
-		const visEttersendeFeatureToggle = visEttersendelse && (visEttersendelse === true);
+		const { originalSoknad, ettersendelser } = this.props;
 		const antallManglendeVedlegg = this.antallManglendeVedlegg();
 		const datoManglendeVedlegg = this.manglendeVedleggDato();
 		const ettersendelseAktivert = this.isEttersendelseAktivert();
```

Comment:
Er ettersendelseAktivert fortsatt i bruk? Det var den som var viktig å fjerne :) 

Hva med informasjon/kommunevalg, er all bruk av dette endepunktet fjernet?

Reply:
Ser at kun kallet som henter kommuner har blitt tatt bort. Dette vil bety at alle ettersendelser blir avvist. Jeg ser på koden nå.

Reply:
Trekker det tilbake. Ser at kallet blir kjørt et annet sted også. Men: Da må vi gjøre ting på gamlemåten i backend,

Reply:
Da bør jirasaken gjenåpnes, for da er løsningen ikke riktig :)

Reply:
Hele poenget var jo å fjerne dette. At featuretoggle-funksjonaliteten fjernes var bare en bonus.

Reply:
@tiafiring Det var en featuretoggling på ettersendelse som jeg ikke fikk med. Har laget en ny pull request som fjerner kode som gjør REST kall til informasjon/kommunevalg 
https://github.com/navikt/soknadsosialhjelp/pull/413